civihr_employee_portal/views/views_export/views_emergency_contacts.inc

```diff
@@ -45,52 +38,176 @@ $handler->display->display_options['fields']['id']['field'] = 'id';
+$handler->display->display_options['fields']['street_address_93']['table'] = 'civicrm_value_emergency_contacts_21';
+$handler->display->display_options['fields']['street_address_93']['field'] = 'street_address_93';
+$handler->display->display_options['fields']['street_address_93']['label'] = '';
+$handler->display->display_options['fields']['street_address_93']['exclude'] = TRUE;
+$handler->display->display_options['fields']['street_address_93']['element_label_colon'] = FALSE;
+/* Field: CiviCRM Custom: Emergency Contacts: Street Address Line 2 */
+$handler->display->display_options['fields']['street_address_line_2_94']['id'] = 'street_address_line_2_94';
+$handler->display->display_options['fields']['street_address_line_2_94']['table'] = 'civicrm_value_emergency_contacts_21';
+$handler->display->display_options['fields']['street_address_line_2_94']['field'] = 'street_address_line_2_94';
+$handler->display->display_options['fields']['street_address_line_2_94']['label'] = '';
+$handler->display->display_options['fields']['street_address_line_2_94']['exclude'] = TRUE;
+$handler->display->display_options['fields']['street_address_line_2_94']['element_label_colon'] = FALSE;
 /* Field: CiviCRM Custom: Emergency Contacts: Relationship with Employee */
 $handler->display->display_options['fields']['relationship_with_employee_83']['id'] = 'relationship_with_employee_83';
 $handler->display->display_options['fields']['relationship_with_employee_83']['table'] = 'civicrm_value_emergency_contacts_21';
 $handler->display->display_options['fields']['relationship_with_employee_83']['field'] = 'relationship_with_employee_83';
+$handler->display->display_options['fields']['relationship_with_employee_83']['label'] = 'Relationship';
+$handler->display->display_options['fields']['relationship_with_employee_83']['exclude'] = TRUE;
+$handler->display->display_options['fields']['relationship_with_employee_83']['element_label_colon'] = FALSE;
+/* Field: CiviCRM Custom: Emergency Contacts: Email */
+$handler->display->display_options['fields']['email_82']['id'] = 'email_82';
+$handler->display->display_options['fields']['email_82']['table'] = 'civicrm_value_emergency_contacts_21';
+$handler->display->display_options['fields']['email_82']['field'] = 'email_82';
+$handler->display->display_options['fields']['email_82']['label'] = 'Primary Email';
+$handler->display->display_options['fields']['email_82']['exclude'] = TRUE;
+$handler->display->display_options['fields']['email_82']['element_label_colon'] = FALSE;
+/* Field: Global: Custom text */
+$handler->display->display_options['fields']['nothing']['id'] = 'nothing';
+$handler->display->display_options['fields']['nothing']['table'] = 'views';
+$handler->display->display_options['fields']['nothing']['field'] = 'nothing';
+$handler->display->display_options['fields']['nothing']['label'] = '';
+$handler->display->display_options['fields']['nothing']['alter']['text'] = '<div class="container">
```

Comment:
This is not maintainable. Why is this necessary and why cant to move this to a html file?